apps/ejabberd/src/ejabberd_c2s.erl

```diff
@@ -139,6 +140,9 @@ broadcast(FsmRef, Type, From, Packet) ->
 stop(FsmRef) ->
     ?GEN_FSM:send_event(FsmRef, closed).
 
+store_info(FsmRef, User, Server, Resource, KV) ->
+    FsmRef ! {store_info, User, Server, Resource, KV, self()}.
```

Comment:
As a somewhat relevant side note, I really don't see the value of the `GEN_FSM` macro. I'd vote for `s/?GEN_FSM/p1_fsm/g`, making the code more obvious and Elvis happy.